src/Sanitize.php

```diff
@@ -166,6 +174,11 @@ public function pass_cache_data($enable_cache = true, $cache_location = './cache
         }
     }
 
+    public function set_http_client(Client $http_client): void
+    {
+        $this->http_client = $http_client;
+    }
+
     public function pass_file_data($file_class = 'SimplePie\File', $timeout = 10, $useragent = '', $force_fsockopen = false)
```

Comment:
Perhaps we can deprecate this method now in favour of `set_http_client`.

Reply:
this would be a topic for #777 where this method will be added.